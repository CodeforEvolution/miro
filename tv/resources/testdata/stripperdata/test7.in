On the front lines of an intergalactic struggle between good and evil, fans young and old will join such favorite characters as Anakin Skywalker, Obi-Wan Kenobi and Padm√© Amidala, along with brand-new heroes like Anakin's padawan learner, Ahsoka. Sinister villains - led by Palpatine, Count Dooku and General Grievous - are poised to rule the galaxy. Stakes are high, and the fate of the "Star Wars" universe rests in the hands of the daring Jedi Knights. Their exploits lead to the action-packed battles and astonishing new revelations that fill "Star Wars: The Clone Wars."<br/>
<br/>
Produced by Lucasfilm Animation, "Star Wars: The Clone Wars" takes audiences on incredible new "Star Wars" adventures, combining the legendary storytelling of Lucasfilm with an eye-popping, signature animation style.<br/>
<br/>
Director: David Filoni<br/>
Screenwriter: Henry Gilroy
