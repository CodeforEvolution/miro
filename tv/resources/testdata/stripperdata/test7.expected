(u'On the front lines of an intergalactic struggle between good and evil, fans young and old will join such favorite characters as Anakin Skywalker, Obi-Wan Kenobi and Padm\xe9 Amidala, along with brand-new heroes like Anakin\'s padawan learner, Ahsoka. Sinister villains - led by Palpatine, Count Dooku and General Grievous - are poised to rule the galaxy. Stakes are high, and the fate of the "Star Wars" universe rests in the hands of the daring Jedi Knights. Their exploits lead to the action-packed battles and astonishing new revelations that fill "Star Wars: The Clone Wars."\nProduced by Lucasfilm Animation, "Star Wars: The Clone Wars" takes audiences on incredible new "Star Wars" adventures, combining the legendary storytelling of Lucasfilm with an eye-popping, signature animation style.\nDirector: David Filoni\nScreenwriter: Henry Gilroy', [])
